lib/util/pragma.js

```diff
@@ -5,19 +5,20 @@
 
 'use strict';
 
-const JSX_ANNOTATION_REGEX = /^\*\s*@jsx\s+([^\s]+)/;
+const JSX_ANNOTATION_REGEX = /@jsx\s+([^\s]+)/;
 // Does not check for reserved keywords or unicode characters
 const JS_IDENTIFIER_REGEX = /^[_$a-zA-Z][_$a-zA-Z0-9]*$/;
 
-
 function getCreateClassFromContext(context) {
   let pragma = 'createReactClass';
   // .eslintrc shared settings (http://eslint.org/docs/user-guide/configuring#adding-shared-settings)
   if (context.settings.react && context.settings.react.createClass) {
     pragma = context.settings.react.createClass;
   }
   if (!JS_IDENTIFIER_REGEX.test(pragma)) {
-    throw new Error(`createClass pragma ${pragma} is not a valid function name`);
+    throw new Error(
+      `createClass pragma ${pragma} is not a valid function name`
+    );
```

Comment:
this looks like `prettier` nonsense; let's revert it please
```suggestion
    throw new Error(`createClass pragma ${pragma} is not a valid function name`);
```